.github/workflows/new-version-check.yml

```diff
@@ -44,7 +44,7 @@ jobs:
             git config user.name "GitHub Actions"
             git config user.email "actions@github.com"
             git commit -m "Bump version to the latest release candidate (${{ env.new_version }})."
-            git push
+            git push stable
```

Comment:
seems like that isn't working as expected, the workflow is now failing on the error `'stable' does not appear to be a git repository`
https://github.com/canonical/firefox-snap/actions/runs/11959914757/job/33342768946